ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_security.schema.yml

```diff
@@ -181,3 +181,84 @@ keys:
                 If specified, one CRL needs to be provided for every certificate in the chain, even if the revocation list in the CRL is empty.
               items:
                 type: str
+      shared_secret_profiles:
+        type: list
+        primary_key: profile_name
+        items:
+          type: dict
+          keys:
+            profile_name:
+              type: str
+            secrets:
+              type: list
+              items:
+                type: dict
+                keys:
+                  name:
+                    type: str
+                  secret:
+                    type: str
+                  lifetime:
+                    type: dict
+                    description: Configure to set lifetime of the secret. This is applicable to both receive and transmit lifetime.
+                    keys:
+                      infinite:
+                        type: bool
+                      start_date_time:
+                        type: str
+                        description: |-
+                          Specify the start date and time of lifetime of the secret.
+                          Formats supported:
+                          1. mm/dd/yyyy hh:mm:ss
+                          2. yyyy-mm-dd hh:mm:ss
+                          e.g 2024/12/20 10:00:00
+                      end_date_time:
+                        type: str
+                        description: |-
+                          Specify the end date and time of lifetime of the secret. End date should be greater than start date time.
+                          Formats supported:
+                          1. mm/dd/yyyy hh:mm:ss
+                          2. yyyy-mm-dd hh:mm:ss
+                          e.g 2024/12/20 10:00:00
+                  receive_lifetime:
+                    type: dict
+                    keys:
+                      infinite:
+                        type: bool
+                      start_date_time:
+                        type: str
+                        description: |-
+                          Specify the start date and time of lifetime of the secret.
+                          Formats supported:
+                          1. mm/dd/yyyy hh:mm:ss
+                          2. yyyy-mm-dd hh:mm:ss
+                          e.g 2024/12/20 10:00:00
+                      end_date_time:
+                        type: str
+                        description: |-
+                          Specify the end date and time of lifetime of the secret. End date should be greater than start date.
+                          Formats supported:
+                          1. mm/dd/yyyy hh:mm:ss
+                          2. yyyy-mm-dd hh:mm:ss
+                          e.g 2024/12/20 10:00:00
```

Comment:
This example does not math any of the given formats.

Reply:
fixed.